api/test/integration/test_security_DELETE_endpoints.tavern.yaml

```diff
@@ -391,29 +391,23 @@ stages:
           total_failed_items: 1
         message: !anystr
 
-  - name: Delete an admin user (wazuh)
+  - name: Delete the current user (Skipped)
```

Comment:
Why dont we have an exception when we try to delete ourselves?

Reply:
It's a design decision, it was easy to implement and it didn't modify any tests nor was it necessary to create an error. We can discuss and add an exception if necessary.